content/en/docs/reference/_index.md

```diff
@@ -66,6 +68,10 @@ client libraries:
 
 * [kube-proxy configuration (v1alpha1)](/docs/reference/config-api/kube-proxy-config.v1alpha1/)
 
+## Config APIs
+
+* [kubelet config (v1beta1)](/docs/reference/config-api/kubelet-config.v1beta1/)
```

Comment:
nit: Should line 69 (kube-proxy configuration) appear under the "Config APIs" heading?

Reply:
This subsection is a little messy when we are merging in the various config APIs. I'm cleaning and rebasing along the way. When the last piece is in, we need to go through this subsection and fix whatever nits we have here.